sky/packages/sky/lib/src/material/drawer.dart

```diff
@@ -141,9 +175,49 @@ class _DrawerRoute extends TransitionRoute {
   }
 
   void _close() {
-    assert(_interactive);
     _performance.fling(velocity: -1.0);
   }
+
+  Widget build(BuildContext context) {
+    _performance.updateVariable(_color);
+    Matrix4 transform = new Matrix4.identity()
+      ..translate((-1.0 + _performance.progress) * (_width ?? 0.0));
+    Widget result = new GestureDetector(
+      onHorizontalDragUpdate: _move,
+      onHorizontalDragEnd: _settle,
+      child: new Stack(<Widget>[
+        new GestureDetector(
```

Comment:
Ok.  Will do in a separate patch.  Currently switching this patch to use Align instead of Transform, which gets rid of the Offstage nonsense.